manager/report_handler.py

```diff
@@ -235,9 +232,9 @@ def handle_get(cls, **kwargs):
                                                    SystemPlatform.advisor_evaluated.is_null(False)) &
                                                   (fn.COALESCE(SystemPlatform.host_type, 'null') != HostType.EDGE))
                          .where(SystemVulnerabilities.rh_account_id == rh_account)
-                         .where((SystemVulnerabilities.mitigation_reason.is_null(True)) &
+                         .where((SystemVulnerabilities.when_mitigated.is_null(True)) &
```

Comment:
Same as the previous query, this query should be left unchanged as it is working correctly. ABNV counts are not needed here.